modules/services/pueue.nix

```
@@ -39,7 +41,7 @@ in {
 
     home.packages = [ cfg.package ];
 
-    xdg.configFile."pueue/pueue.yml".source = configFile;
+    xdg.configFile = mkIf explicitSettings { "pueue/pueue.yml".source = configFile; };
```

I would prefer

```suggestion
    xdg.configFile = mkIf (cfg.settings != { }) { "pueue/pueue.yml".source = configFile; };
```

since we use that everywhere, the `explicitSettings` indirection makes it seem like something special is going on.